io.github.TTimo.GtkRadiant.json

```
@@ -0,0 +1,214 @@
+{
+  "app-id": "io.github.TTimo.GtkRadiant",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "gtkradiant",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--filesystem=host"
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/lib/scons-3.0.1",
+    "/share/pkgconfig",
+    "/share/aclocal",
+    "/man",
+    "/share/man",
+    "/share/gtk-doc",
+    "*.la",
+    "*.a",
+    "/gtkradiant/libjpeg.so*",
+    "/gtkradiant/libpng*",
+    "/gtkradiant/installs/*/.svn"
+  ],
+  "modules": [
+    "shared-modules/python2.7/python-2.7.json",
+    "shared-modules/glu/glu-9.json",
+    "shared-modules/gtk2/gtk2.json",
+    {
+      "name": "pango",
+      "cleanup": [
+          "/share/gir-1.0"
+      ],
+      "sources": [
+          {
+            "type": "archive",
+            "url": "https://download.gnome.org/core/3.26/3.26.2/sources/pango-1.40.13.tar.xz",
+            "sha256": "f84e98db1078772ff4935b40a1629ff82ef0dfdd08d2cbcc0130c8c437857196"
+          }
+      ]
+    },
+    {
+      "name": "pangox",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://archive.ubuntu.com/ubuntu/pool/universe/p/pangox-compat/pangox-compat_0.0.2.orig.tar.xz",
+          "sha256": "552092b3b6c23f47f4beee05495d0f9a153781f62a1c4b7ec53857a37dfce046"
+        }
+      ]
+    },
+    {
+      "name": "gtkglext",
+      "cleanup": [
+        "/lib/gtkglext-1.0/include"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://netcologne.dl.sourceforge.net/project/gtkglext/gtkglext/1.2.0/gtkglext-1.2.0.tar.gz",
+          "sha256": "e5073f3c6b816e7fa67d359d9745a5bb5de94a628ac85f624c992925a46844f9"
+        },
+        {
+          "type": "patch",
+          "path": "patches/gtkglext-1.2.0/01_fix_fontcache_nullderef.diff"
+        },
+        {
+          "type": "patch",
+          "path": "patches/gtkglext-1.2.0/02_fix_gtk-2.20_deprecated_symbols.diff"
+        },
+        {
+          "type": "patch",
+          "path": "patches/gtkglext-1.2.0/libGL.so.1.diff"
+        },
+        {
+          "type": "patch",
+          "path": "patches/gtkglext-1.2.0/03_gdkglext-config-h-installation-path.diff"
+        },
+        {
+          "type": "patch",
+          "path": "patches/gtkglext-1.2.0/glibc2.27-ftbfs.diff"
+        }
```

The config.sub and config.guess in the tarball are too old. This is the fix (untested):

```suggestion
        },
        {
            "type": "shell",
            "commands": [
                "cp -p /usr/share/automake-*/config.{sub,guess} ."
            ]
        }
```